utils/ixdtf/src/parsers/mod.rs

```diff
@@ -388,21 +398,26 @@ impl<'a> Cursor<'a> {
     ///   - Returns a SyntaxError if value is not an ascii digit
     ///   - Returns an AbruptEnd error if cursor ends.
     fn next_digit(&mut self) -> ParserResult<Option<u8>> {
-        let p_digit = self.next_or(ParserError::InvalidEnd)?.to_digit(10);
-        let Some(digit) = p_digit else {
-            return Ok(None);
-        };
-        Ok(Some(digit as u8))
+        // Safety: Char digit with a radix of ten must be in the range of a u8
```

Comment:
Nit: It's not a safety comment since there is no unsafe

```suggestion
        // Note: Char digit with a radix of ten must be in the range of a u8
```